app-backend/db/src/main/scala/com/azavea/rf/database/AccessControlRuleDao.scala

```diff
@@ -39,6 +41,7 @@ object AccessControlRuleDao extends Dao[AccessControlRule] {
   def listedByObject(objectType: ObjectType, objectId: UUID): Dao.QueryBuilder[AccessControlRule] =
     query.filter(fr"object_type = ${objectType}").filter(fr"object_id = ${objectId}")
 
+  @SuppressWarnings(Array("ProductWithSerializableInferred"))
```

Comment:
I don't really understand this warning, but it specifically doesn't like what is happening in:

```scala
val objectDao = ac.objectType match {
```